srcpkgs/haxe/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'haxe'
+pkgname=haxe
+version=4.3.5
+revision=1
+build_style=gnu-makefile
+hostmakedepends="ocaml opam wget tar pkg-config perl rsync which git m4"
+makedepends="pcre2-devel zlib-devel neko-devel mbedtls-devel"
+depends=""
+short_desc="High-level cross-platform programming language"
+maintainer="ohnonotamoth <dilanjohnson@outlook.com>"
+license="MIT"
+homepage="https://haxe.org/"
+distfiles="https://github.com/HaxeFoundation/haxe/archive/refs/tags/${version}.tar.gz"
+checksum=58d79c8c4e01014c00d7e431181f846c42df0324f9d97df752615efa5fcbacbd
+
+pre_build() {
+	git clone https://github.com/HaxeFoundation/haxelib.git extra/haxelib_src
```

Comment:
Please don't `git clone` things like this, grab it in `distfiles` and extract then move it into place.

Reply:
Haxelib has a bunch of git submodules that don't have releases, so this is the only way i could figure out how to do it.